mathesar/api/db/viewsets/records.py

```diff
@@ -145,17 +145,26 @@ def retrieve(self, request, pk=None, table_pk=None):
 
     def create(self, request, table_pk=None):
         table = get_table_or_404(table_pk)
+        primary_key_column_name = None
+        try:
+            primary_key_column_name = table.primary_key_column_name
+        except Exception:
```

Comment:
Please check for a more specific `Exception` here.

Reply:
I've changed it to 'AssertionError' @mathemancer